pkg/prom/registry_test.go

```diff
@@ -22,7 +23,7 @@ func TestRegistryEndpoints(t *testing.T) {
 		namespace        = "fastly"
 		subsystem        = "rt"
 		metricNameFilter = filter.Filter{}
-		registry         = prom.NewRegistry(version, namespace, subsystem, metricNameFilter)
+		registry         = prom.NewRegistry(version, namespace, subsystem, metricNameFilter, realtime.IndividualDatacenter)
```

Comment:
This might not be the right place but I think it would be good to have a test for the AggregatedDatacenter (WithoutDatacenter) option.